Mathlib/Tactic/Ring/Basic.lean

```diff
@@ -282,10 +288,12 @@ inductive Overlap (e : Q($α)) where
   (with nonzero leading coefficient). -/
   | nonzero (_ : Result (ExProd sα) e)
 
-theorem add_overlap_pf (x : R) (e) (pq_pf : a + b = c) :
+-- TODO future! variable {a a₁ a₂ b b₁ b₂ c : R}
```

Comment:
What goes wrong if you do this now?

Reply:
Thanks for pointing out this stray code. It works now (it didn't when `a` and `b` where shadowing some `Q(...)` expressions; these are now all explicit). I've pushed a commit reducing the diff, using this.